Makefile

```
@@ -81,7 +82,31 @@ generate-embeddedyamls: pkg/subctl/operator/common/embeddedyamls/yamls.go
 pkg/subctl/operator/common/embeddedyamls/yamls.go: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go $(shell find deploy/ -name "*.yaml") vendor/modules.txt
 	go generate pkg/subctl/operator/common/embeddedyamls/generate.go
 
-.PHONY: $(TARGETS) test validate build ci clean generate-embeddedyamls operator-image
+# genclientset generates the clientset for the Submariner APIs
+# It needs to be run when the Submariner APIs change
+genclientset:
```

```suggestion
generate-clientset:
```

To be consistent with other target names.

---

Right, I wanted to keep the names as-is in case anything else referred to them, but it seems nothing does.